src/main/resources/avro/variants.avdl

```diff
@@ -117,37 +104,27 @@ particular `Variant`.
 It may include associated information such as quality
 and phasing. For example, a call might assign a probability of 0.32 to
 the occurrence of a SNP named rs1234 in a call set with the name NA12345.
-
-The genotypes described by `Call`s must be consistent with any `AlleleCall`s in
-the same `CallSet`. If a server supports "classic" mode, it must provide `Call`s
-for all `Variant`s that have associated `AlleleCall`s in the `CallSet`.
 */
 record Call {
 
   /**
-  The ID of the call set this variant call belongs to.
+  The name of the call set this variant call belongs to.
   If this field is not present, the ordering of the call sets from a
   `SearchCallSetsRequest` over this `VariantSet` is guaranteed to match
   the ordering of the calls on this `Variant`.
   The number of results will also be the same.
   */
-  union { null, string } callSetId;
+  union { null, string } callSetName = null;
 
   /**
-  The name of the call set this variant call belongs to.
+  The ID of the call set this variant call belongs to.
+
   If this field is not present, the ordering of the call sets from a
   `SearchCallSetsRequest` over this `VariantSet` is guaranteed to match
   the ordering of the calls on this `Variant`.
   The number of results will also be the same.
   */
-  union { null, string } callSetName = null;
-
-  /**
-  The ID of the `Variant` that this `Call` belongs to. Must be set in any `Call`
-  that is not being returned from the server already contained within its
-  `Variant`.
-  */
-  union { null, string } variantId;
```

Comment:
No action required: would be good to reapply this change later (removal of variantId + callsetId).